spec.md

```diff
@@ -936,6 +937,22 @@ module Top:
   node consumer_debug = read(c.out.cref); ; Consumer-side signal
 ```
 
+## Property Types
+
+FIRRTL property types represent information about the circuit that is not
+hardware. This is useful to capture domain specific knowledge and design intent
+alongside the hardware description within the same FIRRTL.
+
+Property types cannot affect hardware functionality or the hardware ABI. They
+cannot be used in any hardware types, including aggregates. They only exist to
```

Comment:
Consider:
```suggestion
cannot be used in any hardware types, including aggregates and references. They only exist to
```

AFAIK this is 100% the case, but how/where to convey that in the spec has some flexibility :+1: .

Reply:
I think `and references` here is a good place, will adopt that.